website/static/js/project-organizer.js

```diff
@@ -364,10 +375,14 @@ var tbOptions = {
                     }
                     tb.filter(event);
                 },
-                onchange: m.withAttr('value', tb.filterText),
+                onchange: function() {
+                    m.withAttr('value', tb.filterText);
```

Comment:
I think onchange is more useful - I am tracking it that way in quicksearch too.  I just want to know that they searched. If I am searching my name, and I type 'D-A-backspace-A-W-N'.   There will be 6 trackClicks fired, D, DA, D, DA, DAW, DAWN.  In early testing I had it onkeyup, and I feel tracking it this way inflates the number beyond where it can be useful.   Onchange here will just send one trackClick.